cypress/integration/userCanSignUp.feature.js

```diff
@@ -0,0 +1,52 @@
+describe("user can sign up", () => {
+  beforeEach(() => {
+    cy.server();
+    cy.visit("/");
+  });
+
+  it("successfully", () => {
+    cy.route({
+      method: "POST",
+      url: "http://localhost:3000/api/auth/",
+      response: "fixture:registration.json",
+      headers: {
+        uid: 'user@mail.com'
+    }
+    });
+    
+    cy.get("#render-signup").click();
+    cy.get("#signup").within(() => {
+      cy.get("#name").type("Mike Shum");
```

Comment:
Are you sending the `name` attribute to your back-end? What happens with it?

Reply:
Our original thought was that we wanted the name to come back as the greeting when signing up and signing in and also we had a thought to use it as a confirmation when picking up the food. But after talking to you and realising it is fairly difficult to make that happen I guess now the name is sent to the backend but does not really have a purpose with the current functionality.